### PR TITLE
feat: add show_tag_keys,show_tag_values,show_field_keys,show_series i…

### PR DESCRIPTION
**Pull request by goyjy**
add show query interface

### CODE REVIEW

File: opengemini_client/client_impl.py [hunk 1]
```diff
@@ -234,3 +235,56 @@ def drop_retention_policy(self, dbname, retention_policy: str):
...
+    def show_field_keys(self, database, command: str) -> List[ValuesResult]:
+        return self._show_with_result_key_value(database, command)
+
+    def show_series(self, database, command: str) -> List[str]:
+        values_result = self._show_with_result_any(database, command)
+        series = []
+        if len(values_result) == 0:
+            return series
+        for value in values_result[0].values:
+            if isinstance(value, str):
```
Comment (zhiheng123) [inline opengemini_client/client_impl.py:288]: Don't we need to handle types other than str here?

**Review by hezhangjian — APPROVED** [review]
LGTM

File: opengemini_client/client_impl.py (hunk 1 above)
Comment (Chenxulin97) [reply opengemini_client/client_impl.py:288]: results don't have other types than str
image

**Review by Chenxulin97 — APPROVED** [review]

File: opengemini_client/client_impl.py (hunk 1 above)
Comment (goyjy) [reply opengemini_client/client_impl.py:288]: This type of query only has str, I want to remove non-str type values

**Pull request merged**